src/libs/state.ts

```diff
@@ -17,67 +17,32 @@ import type { WorkspaceWrapper } from 'src/libs/workspace-utils';
 
 export const routeHandlersStore = atom<unknown[]>([]);
 
-export type TerraUser = {
-  token?: string | undefined;
-  scope?: string | undefined;
-  id?: string | undefined;
-  email?: string | undefined;
-  name?: string | undefined;
-  givenName?: string | undefined;
-  familyName?: string | undefined;
-  imageUrl?: string | undefined;
-  idp?: string | undefined;
-};
+export interface SystemState {
+  termsOfServiceConfig: SamTermsOfServiceConfig;
+}
 
-export type TerraUserProfile = {
-  // TODO: anonymousGroup is here from getProfile from orch
-  // TODO: for future ticket, separate items updated via register/profile (personal info)
-  //  from things that are updated via api/profile/preferences (starred workspaces, notification settings)
-  firstName: string | undefined;
-  lastName: string | undefined;
-  institute: string | undefined;
-  contactEmail: string | undefined;
-  title: string | undefined;
-  department: string | undefined;
-  interestInTerra: string | undefined;
-  programLocationCity?: string;
-  programLocationState?: string;
-  programLocationCountry?: string;
-  researchArea?: string;
-  starredWorkspaces?: string;
-};
+/**
+This store is for keeping track of static system properties
+which are not expected to change during the apps lifecycle
+ */
+export const systemStore: Atom<SystemState> = atom<SystemState>({
```

Comment:
We already have configStore / getConfig for things like this. Would it make sense to add ToS configuration there instead of creating a new store?